Latex/vorlage/vorlage.tex

```diff
@@ -1,13 +1,13 @@
 %! LaTeX Vorlage
-\documentclass[12pt, fleqn, captions=nooneline, titlepage, footsepline, headsepline, toc=sectionentrywithdots, listof=entryprefix, bibliography=totoc, parskip=half-]{scrartcl}
+\documentclass[12pt, fleqn, captions=nooneline, titlepage, footsepline, headsepline, toc=chapterentrywithdots, listof=entryprefix, bibliography=totoc, parskip=half-]{scrreprt}
 
 \usepackage{silence} %unnötige Warnungen unterdrücken
 \WarningFilter{latex}{You have requested}
-\WarningFilter{scrlayer-scrpage}{\headheight to low}
-\WarningFilter{scrlayer-scrpage}{\footheight to low}
-\WarningFilter{scrlayer-scrpage}{Very small head height detected}
-\WarningFilter{fvextra}{}
-\WarningFilter{lineno}{}
+% \WarningFilter{scrlayer-scrpage}{\headheight to low}
+% \WarningFilter{scrlayer-scrpage}{\footheight to low}
+% \WarningFilter{scrlayer-scrpage}{Very small head height detected}
+% \WarningFilter{fvextra}{} % was caused by loading csquotes before minted (which loads fvextra)
+% \WarningFilter{lineno}{}
```

Comment:
wollen wir die auskommentierten ganz rausnehmen?
```suggestion
```

Reply:
ich hab zwar inzwischen das ganze auch mal mit PM4 getestet aber ich vertrau der ruhe nich... ich hätte die gern noch zur einfachen reaktivierung da... können wir wegen mir auch löschen aber dann muss ich erst in git schauen wenn was davon doch zurück kommt

Reply:
Als BAckup erstmal stehen lassen...